donkeycarManagerFront/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Donkeycar front build
+FROM node:18 AS build-stage
+COPY package*.json ./
+RUN npm install       
+COPY . .
+WORKDIR ./donkeycarManagerFront
+RUN  npm install @vue/cli-service --save-dev
+RUN npm run build
```

Comment:
It fail, seems to work only with the flag `--legacy-peer-deps`, this means that the lock file needs to be updated / change.

```
docker build -t benvii/donkeycarlph-front .
Sending build context to Docker daemon  173.7MB
Step 1/11 : FROM node:18 AS build-stage
18: Pulling from library/node
bd73737482dd: Pull complete 
6710592d62aa: Pull complete 
75256935197e: Pull complete 
c1e5026c6457: Pull complete 
f2e4b4cbd0b8: Pull complete 
06cfeca21bcd: Pull complete 
cb6259d61ade: Pull complete 
e7ad8f1516a1: Pull complete 
Digest: sha256:8cdf7234449f35e579f7491137807ddb3a089f028862f7ab69af437cc9f47ff1
Status: Downloaded newer image for node:18
 ---> 78b037dbb659
Step 2/11 : COPY package*.json ./
 ---> d1b541b5545c
Step 3/11 : RUN npm install
 ---> Running in 280ec8658419
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @vue/eslint-config-standard@6.1.0
npm ERR! Found: eslint-plugin-vue@8.7.1
npm ERR! node_modules/eslint-plugin-vue
npm ERR!   dev eslint-plugin-vue@"^8.0.3" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint-plugin-vue@"^7.0.0" from @vue/eslint-config-standard@6.1.0
npm ERR! node_modules/@vue/eslint-config-standard
npm ERR!   dev @vue/eslint-config-standard@"^6.1.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: eslint-plugin-vue@7.20.0
npm ERR! node_modules/eslint-plugin-vue
npm ERR!   peer eslint-plugin-vue@"^7.0.0" from @vue/eslint-config-standard@6.1.0
npm ERR!   node_modules/@vue/eslint-config-standard
npm ERR!     dev @vue/eslint-config-standard@"^6.1.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /root/.npm/_logs/2023-06-10T16_12_07_615Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2023-06-10T16_12_07_615Z-debug-0.log
The command '/bin/sh -c npm install' returned a non-zero code: 1

```

Reply:
Fixe here [ad96fa4](https://github.com/mdl29/donkeycarLPH/pull/259/commits/ad96fa48071154afea43dbdba608c982b84883b7)